website/docs/cert-manager.md

```diff
@@ -0,0 +1,134 @@
+
+### Issuing a CA certificate
+
+Let's create the CA certificate for our example :
+
+```yaml
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: gatekeeper-selfsigning-issuer
+  namespace: gatekeeper-system
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: gatekeeper-ca-certificate
+  namespace: gatekeeper-system
+spec:
+  secretName: gatekeeper-ca-certificate
+  isCA: true
+  commonName: gatekeeper-ca
+  issuerRef:
+    name: gatekeeper-selfsigning-issuer
+```
+
+If successful, you should have a valid certificate and an associated secret :
+```
+$ kubectl -n gatekeeper-system get cert gatekeeper-ca-certificate
+NAME                        READY   SECRET                      AGE
+gatekeeper-ca-certificate   True    gatekeeper-ca-certificate   3h6m
+$ kubectl -n gatekeeper-system get secret gatekeeper-ca-certificate
+NAME                        TYPE                DATA   AGE
+gatekeeper-ca-certificate   kubernetes.io/tls   3      3h6m
+```
+
+You can check the content of the CA certificate as following :
+```
+kubectl -n gatekeeper-system get secret gatekeeper-ca-certificate -o go-template='{{index .data "ca.crt"}}' | base64 -d | openssl x509 -noout -text
+```
+
+In the real world, there is no reason why you would generate a self signed CA certificate. Tis is for te purpose of the example. The CA was most probably issued by a trusted authority at the organisation level.
+
+### TLS certificate
+
+Let's create now a TLS certificate for the service `gatekeeper-webhook-service` from our authority :
+
+```yaml
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: gatekeeper-ca-issuer
+  namespace: gatekeeper-system
+spec:
+  ca:
+    secretName: gatekeeper-ca-certificate
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: gatekeeper-certificate
+  namespace: gatekeeper-system
+spec:
+  secretName: gatekeeper-certificate
+  commonName: gatekeeper-webhook-service.gatekeeper-system.svc
+  issuerRef:
+    name: gatekeeper-ca-issuer
+    name: gatekeeper-selfsigning-issuer
+    kind: Issuer
+```
+
+If successful, you should have a valid certificate and a related TLS secret similar to :
+```
+$ kubectl -n gatekeeper-system get cert gatekeeper-certificate
+NAME                     READY   SECRET                   AGE
+gatekeeper-certificate   True    gatekeeper-certificate   3h9m
+$ kubectl -n gatekeeper-system get secret gatekeeper-certificate
+NAME                     TYPE                DATA   AGE
+gatekeeper-certificate   kubernetes.io/tls   3      3h8m
+```
+
+You can check the content of the generated certificate :
+```
+kubectl -n gatekeeper-system get secret gatekeeper-certificate -o go-template='{{index .data "tls.crt"}}' | base64 -d | openssl x509 -noout -text
+```
+
+## Configure the helm chart parameters
```

Comment:
This assumes the use of Helm, which is not necessarily true. We should have a section that highlights both the helm chart parameters and the manifest.

E.g. for helm, set these fields, for the manifest, these flags must be set, and the certificates are read from these other mount points"

Context for how GK reads certs:

https://github.com/open-policy-agent/gatekeeper/blob/bf94eb335918f8806571e28d01fb1c26a9179b2d/main.go#L98-L99

Reply:
It makes sense